.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "htmlbook"]
 	path = htmlbook
 	url = https://github.com/RussTedrake/htmlbook
+[submodule "underactuated"]
```

Comment:
PlanarSceneGraphVisualizer is in drake master now.  i was hoping that I wouldn't need to include underactuated here?  I will test.

Reply:
looks like it is (only?) blocking on https://github.com/RobotLocomotion/drake/pull/12349 . i think we should get this PR in without underactuated (or trimesh, etc) and just don't add the planar_iiwa test to CI for now.  

btw -- thank you so much for doing all this!

Reply:
Done.